{{cookiecutter.project_slug}}/dev.yml

```diff
@@ -19,7 +19,8 @@ services:
       dockerfile: ./compose/django/Dockerfile-dev
     command: /start-dev.sh
     depends_on:
-      - postgres
+      - postgres{% if cookiecutter.use_mailhog == 'y' %}
```

Comment:
Github email fooled me with wrong diff. So my comment was unnecessary because you did not touch those lines. sorry.

in relation of Jinja2 sintax, you could use `{%-` instead of `{%` to suppress new line before and after `if`


```yml
    depends_on:
      - postgres
{%- if cookiecutter.use_mailhog == 'y' %}
      - mailhog
{%- endif %}
    env_file:
      - dev.env
```

the rendered result is

```yml
    depends_on:
      - postgres
      - mailhog
    env_file:
      - dev.env
```

if you use `{%`, the rendered result is will be

```yml
    depends_on:
      - postgres

      - mailhog

    env_file:
      - dev.env
```

`-%}` It can also be useful in some cases.

To explore syntax options a bit more, this site may be useful http://jinja2test.tk/